### PR TITLE
Add GeoPoint support to explorer.

### PR DESCRIPTION
**Pull request by 0candy**
Fix https://github.com/strongloop/loopback-component-explorer/issues/136

@bajtos  Please review.

### CODE REVIEW

File: lib/specgen/type-registry.js [hunk 1]
```diff
@@ -10,7 +10,8 @@ function TypeRegistry() {
 
   this.register('x-any', { properties: {} });
   this.register('ObjectID', { type: 'string', pattern: '^[a-fA-F\\d]{24}$' });
-  // TODO - register GeoPoint and other built-in LoopBack types
+  this.register('GeoPoint', { properties: {} });
```
Comment (raymondfeng) [inline lib/specgen/type-registry.js]: Maybe the following is better:
```js
this.register('GeoPoint', { properties: {lat: 'number', lat: 'number'} });
```

Comment (bajtos) [inline lib/specgen/type-registry.js]: Yes please, let's include the definition of properties.

File: lib/specgen/type-registry.js [hunk 2]
```diff
@@ -10,7 +10,8 @@ function TypeRegistry() {
...
+  // TODO - register other built-in LoopBack types
```
Comment (bajtos) [inline lib/specgen/type-registry.js]: I think there are no more built-in types to register, this comment can be removed.

File: lib/specgen/type-registry.js (hunk 1 above)
Comment (0candy) [inline lib/specgen/type-registry.js]: Should be lat & **lng** I assume?
  this.register('GeoPoint', { properties: { lat: 'number', **lng**: 'number' } });

**Comment by 0candy** [comment]
@raymondfeng  or @bajtos Please review.

**Comment by bajtos** [comment]
LGTM. Could you please squash the commits?

**Pull request merged**